tools/qt-builder/README.md

```diff
@@ -288,44 +285,40 @@ Zip up this directory and upload it to Backtrace or other crash log handlng tool
 
 
 ### Mac  
-**TODO: Update this section for Qt 5.15.2**  
 
 #### Preparing source files  
-git clone --recursive git://code.qt.io/qt/qt5.git -b 5.12.3 --single-branch    
-  
-*  Copy the **patches** folder to qt5  
-*   Apply the patches to Qt  
-`cd qt5`  
-`git apply --ignore-space-change --ignore-whitespace patches/aec.patch`  
-`git apply --ignore-space-change --ignore-whitespace patches/qtscript-crash-fix.patch`  
-`git apply --ignore-space-change --ignore-whitespace patches/mac-web-video.patch`  
-`cd ..`  
+git clone --recursive git://code.qt.io/qt/qt5.git -b 5.15.2 --single-branch    
+
+*  If you are compiling with MacOSX11.1.SDK or greater, edit qt5/qtwebengine/src/3rdparty/chromium/build/mac/find_sdk.py line 91 and replace "MacOSX(10" with "MacOSX(11".
 
 #### Configuring
 `mkdir qt5-install`  
 `mkdir qt5-build`  
 `cd ../qt5-build`  
 
-`../qt5/configure -force-debug-info -opensource -confirm-license -qt-zlib -qt-libjpeg -qt-libpng -qt-freetype -qt-pcre -qt-harfbuzz -nomake examples -nomake tests -skip qttranslations -skip qtserialport -skip qt3d -skip qtlocation -skip qtwayland -skip qtsensors -skip qtgamepad -skip qtspeech -skip qtcharts -skip qtx11extras -skip qtmacextras -skip qtvirtualkeyboard -skip qtpurchasing -skip qtdatavis3d -no-warnings-are-errors  -no-pch -prefix ../qt5-install`  
-**TODO: Remove `-skip qtspeech`**
+`../qt5/configure -force-debug-info -opensource -confirm-license -qt-zlib -qt-libjpeg -qt-libpng -qt-freetype -qt-pcre -qt-harfbuzz -nomake examples -nomake tests -skip qttranslations -skip qtserialport -skip qt3d -skip qtlocation -skip qtwayland -skip qtsensors -skip qtgamepad -skip qtcharts -skip qtx11extras -skip qtmacextras -skip qtvirtualkeyboard -skip qtpurchasing -skip qtdatavis3d -no-warnings-are-errors  -no-pch -prefix ../qt5-install`  
 
 #### Make
 `make`  
 `make install`  
 
 #### Fixing
 1.  The *.prl* files have an absolute path that needs to be removed (see http://www.linuxfromscratch.org/blfs/view/stable-systemd/x/qtwebengine.html)  
-`cd ../qt5-install`  
+`cd` to the `qt5-install directory`  
 `find . -name \*.prl -exec sed -i -e '/^QMAKE_PRL_BUILD_DIR/d' {} \;`  
 `cd ..`  
-1.   Copy *qt.conf* to *qt5-install\bin*
+1.   Note: you may have additional files in qt5-install/lib and qt5-install/lib/pkg/pkgconfig that have your local build absolute path included.  Optionally you can fix these as well, but it will not effect the build if left alone.
+
+Add a *qt.conf* file.  
+1. Copy the file *qt5-build\qtbase\bin\qt.conf* to *qt5-install\bin*
+1. Edit the *qt.conf* file: replace all absolute URLs with relative URLs (beginning with .. or .)
```

Comment:
You are using `1.` four times in the same section

Reply:
I followed same markup notation used in windows section, which renders correctly although in source looks to be 1. multiple times.  Please refer to original document source before my change.